Message text view

ryang1428 opened the pull request:
- Now using a `UITextView` instead of a `UILabel` for the alerts message
- Supports iOS 7 and 8

ryang1428 committed:
Made the message a UITextView
- Handling max height so it only expands so much until it becomes scrollable
- Removing textview insets

ryang1428 committed:
Merge branch 'master' of github.com:urbn/URBNAlert into MessageTextView

ryang1428 committed:
Merge branch 'master' of github.com:urbn/URBNAlert into MessageTextView

ryang1428 committed:
iOS 7 bug fix

codecaffeine commented on Pod/Classes/URBNAlertView.m:
```diff
@@ -148,7 +200,7 @@ - (UILabel *)titleLabel {
         _titleLabel = [UILabel new];
         _titleLabel.numberOfLines = 2;
-        _titleLabel.textAlignment = self.alertStyler.titleAlignment;
+        _titleLabel.textAlignment = NSTextAlignmentCenter;
```
Why not use `self.alertStyler.titleAlignment`?

ryang1428 replied:
opps.. yeah it should be that

ryang1428 replied:
fixed

ryang1428 committed:
fix

codecaffeine committed:
Some of these properties weren’t being copied.

I tried going through 1-by-1 to see which ones weren’t, but it was 
easier to just copy-paste the property list from the interface.

codecaffeine committed:
Merge branch 'MessageTextView' of github.com:urbn/URBNAlert into MessageTextView

codecaffeine committed:
Fade Blur View

Just to be a bit prettier 😁
